SparkPost/matchbox
# FE-813 Update Checkbox and Checkbox.Group Components

### What Changed
Restyles Checkbox
- #355 - Checkbox `id` prop is now required
- #355 - Checkbox now properly hides a screen reader accessible label
- #355 - Checkbox now properly uses aria-describedby to link to its helptext and error

<img width="258" alt="Screen Shot 2020-03-17 at 3 07 16 PM" src="https://user-images.githubusercontent.com/3903325/76960174-4c04a000-68f1-11ea-80cc-a9e5fd06a8cd.png">

### How To Test or Verify
- Run storybook with `npm run start:storybook`
- Visit & verify Checkbox stories

### PR Checklist
